BayesianTools/R/BayesianSetupGenerateParallel.R

```diff
@@ -2,14 +2,26 @@
 #' 
 #' @author Florian Hartig
 #' @param fun function to be changed to parallel execution
-#' @param parallel should a parallel R cluster be used or not. If set to T, cores will be detected automatically and n-1 of the available n cores of the machine will be used. Alternatively, you can set the number of cores used by hand
-#' @param parallelOptions list containing three lists. First "packages" determines the R packages necessary to run the likelihood function. Second "variables" the objects in the global environment needed to run the likelihood function and third "dlls" the DLLs needed to run the likelihood function (see Details).
-#' @note Can also be used to make functions compatible with library sensitivity
-#' @details For parallelization, option T means that an automatic parallelization via R is attempted, or "external", in which case it is assumed that the likelihood is already parallelized. In this case it needs to accept a matrix with parameters as columns.
-#' Further you can specify the packages, objects and DLLs that are exported to the cluster. 
-#' By default a copy of your workspace is exported. However, depending on your workspace this can be very inefficient. 
-#'
-#' Alternatively you can specify the environments and packages in the likelihood function (e.g. BayesianTools::VSEM() instead of VSEM()).
+#' @param parallel should a parallel R cluster be used? If set to T, machine will
```

Comment:
With the line breaks, I can't see which words have changed in the diff because all seems to be changed - could you change back to how it was before?